student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(id, age, classroom, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission, name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] the id should be created in the initialize method of the person class it should not be set by the user, have a look at [this lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more details.